source.extension.vsixmanifest

```diff
@@ -1,23 +1,29 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
-  <Metadata>
-    <Identity Id="CatchTestAdapter.Klaus Beyer.f56db3ea-257e-4f65-a52c-a6160aa8e329" Version="1.0" Language="en-US" Publisher="Klaus Beyer" />
-    <DisplayName>CatchTestAdapter</DisplayName>
-    <Description xml:space="preserve">Test Adapter for the Catch C++ unit test framework</Description>
-    <License>LICENSE.txt</License>
-    <Icon>Resources\VSPackage1.ico</Icon>
-    <Tags>Unit Test, C++, Visual Studio 2015</Tags>
-  </Metadata>
-  <Installation>
-    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[14.0]" />
-  </Installation>
-  <Dependencies>
-    <Dependency Id="Microsoft.Framework.NDP" DisplayName="Microsoft .NET Framework" d:Source="Manual" Version="[4.5,)" />
-    <Dependency Id="Microsoft.VisualStudio.MPF.14.0" DisplayName="Visual Studio MPF 14.0" d:Source="Installed" Version="[14.0]" />
-  </Dependencies>
-  <Assets>
-    <Asset Type="Microsoft.VisualStudio.VsPackage" d:Source="Project" d:ProjectName="%CurrentProject%" Path="|%CurrentProject%;PkgdefProjectOutputGroup|" />
-    <Asset Type="UnitTestExtension" d:Source="Project" d:ProjectName="TestAdapter" Path="|TestAdapter|" />
-    <Asset Type="Microsoft.VisualStudio.MefComponent" d:Source="Project" d:ProjectName="TestAdapter" Path="|TestAdapter|" />
-  </Assets>
+    <Metadata>
+        <Identity Id="CatchTestAdapter.Klaus Beyer.f56db3ea-257e-4f65-a52c-a6160aa8e329" Version="1.0" Language="en-US" Publisher="Klaus Beyer" />
+        <DisplayName>CatchTestAdapter</DisplayName>
+        <Description xml:space="preserve">Test Adapter for the Catch C++ unit test framework</Description>
+        <License>LICENSE.txt</License>
+        <Icon>Resources\VSPackage1.ico</Icon>
+        <Tags>Unit Test, C++, Visual Studio 2015</Tags>
+    </Metadata>
+    <Installation>
+        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[14.0,16.0)" />
+        <InstallationTarget Version="[14.0,16.0)" Id="Microsoft.VisualStudio.Pro" />
+        <InstallationTarget Version="[14.0,16.0)" Id="Microsoft.VisualStudio.Enterprise" />
+    </Installation>
+    <Dependencies>
+        <Dependency Id="Microsoft.Framework.NDP" DisplayName="Microsoft .NET Framework" d:Source="Manual" Version="[4.5,)" />
```

Comment:
4.6.1 should be added.

Reply:
You are right. It is better to stay with 4.5. It makes no sense to switch to a newer one if not needed.